### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mladenjordanov**
I will add more info about changing pellets and gun later

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,34 @@
...
-In the Arduino IDE go to File | Preferences and add 
+## Required Board Definitions
 
-https://m5stack.oss-cn-shenzhen.aliyuncs.com/resource/arduino/package_m5stack_index.json 
+Before you can use this code, you need to have the M5 Stack board definitions installed in the Arduino IDE. To do this, follow these steps:
 
-To the Additional board manager URLS
+1. In the Arduino IDE, go to `File` > `Preferences`
+2. Add `https://m5stack.oss-cn-shenzhen.aliyuncs.com/resource/arduino/package_m5stack_index.json` to the `Additional board manager URLS` field
+3. Go to `Tools` > `Board` > `Boards manager`
```
Comment (GlovePuppet) [inline README.md:13]: You may need to restart the Arduino IDE for this change to take effect

File: README.md [hunk 2]
```diff
@@ -1,39 +1,76 @@
...
-https://m5stack.oss-cn-shenzhen.aliyuncs.com/resource/arduino/package_m5stack_index.json 
+# Setting up the Arduino IDE environment for M5StickC
 
-To the Additional board manager URLS
+## Step 1: Download and install the Arduino IDE
+1. Download the latest version of the Arduino IDE from the official website: https://www.arduino.cc/en/software
+2. Install the Arduino IDE on your computer
 
-Go to Tools | Board | Boards manager 
+## Step 2: Add the ESP32 board manager URL
```
Comment (DaystateRebel) [inline README.md:15]: I think this is the URL
https://m5stack.oss-cn-shenzhen.aliyuncs.com/resource/arduino/package_m5stack_index.json

I think we need to restart the IDE after adding this

File: README.md [hunk 3]
```diff
@@ -1,39 +1,76 @@
...
+1. Start the Arduino IDE
+2. Go to File -> Preferences
+3. In the "Additional Boards Manager URLs" field, add the following URL: 'https://dl.espressif.com/dl/package_esp32_index.json'
 
-Search for and install "M5Stack" 
 
-Go to Tools | Board
+## Step 3: Install the ESP32 package
```
Comment (DaystateRebel) [inline README.md:22]: I don't think we need the esp32 but I think we do need the M5Stack package

File: README.md [hunk 4]
```diff
@@ -1,39 +1,76 @@
...
+1. Go to Tools -> Board -> Boards Manager
+2. Search for "esp32"
+3. Install the "esp32 by Espressif Systems" package
 
-Select M5Stick-C-Plus
+## Step 4: Select the ESP32 Wrover Module board
```
Comment (DaystateRebel) [inline README.md:27]: Select the M5Stick-C-Plus

Tools | Board | M5Stack Arduino | M5Stick-C-Plus

File: README.md [hunk 5]
```diff
@@ -1,39 +1,76 @@
...
-## Required Libraries 
+## Step 5: Connect the M5StickC to your computer
+1. Connect your M5StickC to your computer using a USB-C cable
 
-### M5StickCPlus
+## Step 6: Select the correct port
+1. Go to Tools -> Port
+2. Select the port that your M5StickC is connected to
+   - On Windows: "COMx"
+   - On Mac and Linux: "/dev/cu.usbserial-xxxxx"
```
Comment (DaystateRebel) [inline README.md:38]: On linux /dev/ttyUSBx

File: README.md [hunk 6]
```diff
@@ -1,39 +1,76 @@
...
+2. Add `https://m5stack.oss-cn-shenzhen.aliyuncs.com/resource/arduino/package_m5stack_index.json` to the `Additional board manager URLS` field
+3. Go to `Tools` > `Board` > `Boards manager`
+4. Search for and install "M5Stack"
+5. Go to `Tools` > `Board`
+6. Select `M5Stick-C-Plus`
+
+## Required Libraries
 
-Got to Sketch | Include Library | Add .Zip Library
+This project uses the M5StickCPlus and OpenFontRender libraries. To install these libraries, follow these steps:
```
Comment (DaystateRebel) [inline README.md]: Also, M5-ENV

https://github.com/m5stack/M5Unit-ENV/tree/master/src

File: README.md [hunk 7]
```diff
@@ -1,39 +1,76 @@
...
+## Introduction
 
-## Required Board definitions: 
+This repository contains the FX Chrony display code for the M5 Stick C Plus. The FX Chrony display is a simple and effective way to keep track of your shooting data in real-time. With this project, you can quickly and easily get started with using the M5 Stick C Plus to display important shooting information such as your shot velocity, shot count, and more.
 
-In the Arduino IDE go to File | Preferences and add 
+Instructions to start with Arduino IDE 'https://docs.m5stack.com/en/quick_start/m5stickc/arduino'
 
-https://m5stack.oss-cn-shenzhen.aliyuncs.com/resource/arduino/package_m5stack_index.json 
+# Setting up the Arduino IDE environment for M5StickC
```
Comment (DaystateRebel) [inline README.md:9]: Maybe we can save some work by linking to this for initial setup?

https://docs.m5stack.com/en/quick_start/m5stickc_plus/arduino

**Pull request merged**